### PR TITLE
DBZ-1906

### PR DESCRIPTION
**Pull request by gunnarmorling**
https://issues.redhat.com/browse/DBZ-1906

### CODE REVIEW

**Comment by gunnarmorling** [comment]
@jpechane, @Naros, could either of you review and merge? Should also be backported to 1.0 and 1.1. Thanks!

File: documentation/modules/ROOT/pages/modules/tutorial/p_viewing-create-event.adoc [hunk 1]
```diff
@@ -334,7 +334,7 @@ However, the Kafka topic containing all of the events for a single table might h
 
 The JSON converter includes the key and value schemas in every message,
 so it does produce very verbose events.
-Alternatively, you can use the link:http://docs.confluent.io/3.1.2/schema-registry/docs/index.html[Avro converter], which results in far smaller event messages.
+Alternatively, you can use xref:configuration/avro.adoc[Apache Avro] as a serialization format, which results in far smaller event messages.
```
Comment (Naros) [inline documentation/modules/ROOT/pages/modules/tutorial/p_viewing-create-event.adoc]: I believe we probably should add the cdc-product conditional here as any other prior uses in the downstream docs that reference `configuration/avro.adoc` are wrapped using `ifndef::cdc-product[]`.

File: documentation/antora.yml [hunk 2]
```diff
@@ -25,3 +25,4 @@ asciidoc:
     link-oracle-plugin-snapshot: 'https://oss.sonatype.org/service/local/artifact/maven/redirect?r=snapshots&g=io.debezium&a=debezium-connector-oracle&v=LATEST&c=plugin&e=tar.gz'
     link-db2-plugin-snapshot: 'https://oss.sonatype.org/service/local/artifact/maven/redirect?r=snapshots&g=io.debezium&a=debezium-connector-db2&v=LATEST&c=plugin&e=tar.gz'
     link-cassandra-plugin-snapshot: 'https://oss.sonatype.org/service/local/artifact/maven/redirect?r=snapshots&g=io.debezium&a=debezium-connector-cassandra&v=LATEST&c=plugin&e=tar.gz'
+    link-kafka-docs: 'https://kafka.apache.org/documentation'
```
Comment (Naros) [inline documentation/antora.yml]: In my discussion with the downstream docs team, the `antora.yml` file is not consumed downstream.  This is what has allowed us to add our own attributes here and they are able to easily override them downstream.  We most likely need to coordinate with them so they also add this attribute to their build process as well.

Comment (gunnarmorling) [reply documentation/antora.yml]: Ah, I see. @bhardesty, can you take a look at this PR then, too? IIUC, you'd need to add a definition of this variable in the downstream docs build, too. Thanks!

Comment (gunnarmorling) [reply documentation/antora.yml]: Summoning @fbolton, too. I'm about to merge this PR, so it'd mean you'd have to add that variable to the downstream docs build environment, too. Is that ok?

File: documentation/modules/ROOT/pages/modules/tutorial/p_viewing-create-event.adoc (hunk 1 above)
Comment (gunnarmorling) [reply documentation/modules/ROOT/pages/modules/tutorial/p_viewing-create-event.adoc]: Added.

File: documentation/antora.yml (hunk 2 above)
Comment (bhardesty) [reply documentation/antora.yml]: I just added the attribute for the downstream docs, so they should be good now.

**Comment by gunnarmorling** [comment]
Thanks, Ben!

>

**Pull request merged**